packages/std/src/mock.rs

```diff
@@ -235,39 +235,68 @@ fn mock_ibc_channel(my_channel_id: &str, order: IbcOrder, version: &str) -> IbcC
         },
         order,
         version: version.to_string(),
-        counterparty_version: Some(version.to_string()),
         connection_id: "connection-2".to_string(),
     }
 }
 
 #[cfg(feature = "stargate")]
-/// Creates a IbcChannelOpenMsg for testing ibc_channel_open.
-pub fn mock_ibc_channel_open(
+/// Creates a IbcChannelOpenMsg::OpenInit for testing ibc_channel_open.
```

Comment:
Let's move the doc comments above the attributes as suggested in https://github.com/rust-dev-tools/fmt-rfcs/issues/72, https://github.com/rust-lang/rustfmt/issues/3303, https://github.com/rust-lang/rustfmt/issues/3744